Makefile

```diff
@@ -539,38 +539,21 @@ nuke: clean clean-docker
 setup-mac:
 	echo $$(boot2docker ip 2> /dev/null) dockerhost | sudo tee -a /etc/hosts
 
+NON_ENTERPRISE_PACKAGES=./api/... ./api4/... ./app/... ./cmd/... ./einterfaces/... ./imports/... ./jobs/... \
```

Comment:
Hmm... I thought I had responded, but I guess I never submitted it.

This change was not unrelated. This had to be changed in some way to get vet running against the new package. I very much disagree that this should have been a separate PR and think there might be discussions to be had about opportunistic refactoring.